legal-api/src/legal_api/models/business.py

```diff
@@ -48,23 +48,46 @@ class Business(db.Model):  # pylint: disable=too-many-instance-attributes
     class LegalTypes(Enum):
         """Render an Enum of the Business Legal Types."""
 
-        COOP = 'CP'
-        BCOMP = 'BEN'
-        COMP = 'BC'
+        COOP = 'CP'  # aka COOPERATIVE in namex
+        BCOMP = 'BEN'  # aka BENEFIT_COMPANY in namex
+        COMP = 'BC'  # aka CORPORATION in namex
         CONTINUE_IN = 'C'
         CO_1860 = 'QA'
         CO_1862 = 'QB'
         CO_1878 = 'QC'
         CO_1890 = 'QD'
         CO_1897 = 'QE'
-        BC_ULC_COMPANY = 'ULC'  # BC Unlimited Liability Company
+        BC_ULC_COMPANY = 'ULC'
         ULC_CONTINUE_IN = 'CUL'
         ULC_CO_1860 = 'UQA'
         ULC_CO_1862 = 'UQB'
         ULC_CO_1878 = 'UQC'
         ULC_CO_1890 = 'UQD'
         ULC_CO_1897 = 'UQE'
         BC_CCC = 'CC'  # BC Community Contribution Company
+        # *** the following are from namex project ***
+        # CORPORATION = 'CR'  # use COMP instead
+        UNLIMITED_LIABILITY_COMPANY = 'UL'
+        SOLE_PROPRIETORSHIP = 'FR'
+        GENERAL_PARTNERSHIP = 'GP'
+        DOING_BUSINESS_AS = 'DBA'
```

Comment:
DBA is not a business type.

This was a dump of all the business types from colin https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/business.json#L20
and bcgov/entity#7731 already defined all the corp types

Reply:
Thanks. I rebased to the 7731 changes. (I still have a few types I don't know how to map across - see comment in ticket.)

Reply:
Hey, should I sort these? By key or value?

Reply:
I commented out the entity types that Lear doesn't yet support.

(I think this is better than deleting the code as we may want to enable these some time.)